luaui/Widgets/gui_info.lua

```diff
@@ -371,28 +371,38 @@ local function refreshUnitInfo()
 					end
 
 				elseif weaponDef.customParams then
-					if weaponDef.customParams.cluster then -- Bullets that explode into other, smaller bullets
+					-- Handle weaponDefs that spawn extra projectiles or explosions.
+					if weaponDef.customParams.cluster then
 						unitExempt = true
 						calculateClusterDPS(weaponDef, weaponDef.damages[0])
-					elseif weaponDef.customParams.speceffect == "split" then -- Bullets that split into other, smaller bullets
+					elseif weaponDef.customParams.speceffect == "split" then
 						unitExempt = true
 						local splitd = WeaponDefNames[weaponDef.customParams.def].damages[0]
 						local splitn = weaponDef.customParams.number or 1
-						calculateWeaponDPS(weaponDef, splitd * splitn)
-					--end
-
+						calculateWeaponDPS(weaponDef, weaponDef.damages[0] * splitd * splitn)
 					elseif weaponDef.customParams.spark_basedamage then -- Lightning
 						unitExempt = true
 						local forkd = weaponDef.customParams.spark_forkdamage
 						local forkn = weaponDef.customParams.spark_maxunits or 1
 						calculateWeaponDPS(weaponDef, weaponDef.damages[0] * (1 + forkd * forkn))
-						if unitExempt and weaponDef.paralyzer then -- DPS => EMP
-							unitDefInfo[unitDefID].minemp = unitDefInfo[unitDefID].mindps
-							unitDefInfo[unitDefID].maxemp = unitDefInfo[unitDefID].maxdps
-							unitDefInfo[unitDefID].mindps = nil
-							unitDefInfo[unitDefID].maxdps = nil
+					elseif weaponDef.customParams.overpen_exp_def then
+						local wepDamage = weaponDef.damages[0]
+						local expDamage = WeaponDefNames[weaponDef.customParams.overpen_exp_def].damages[0]
+						if string.find(weaponDef.name, "nuclear_missile") then -- ignore the "impact" damage to fighters, etc.
```

Comment:
More likely, this will be removed. I was using the overpen gadget to test nukes breaking through fighter walls -- which does work but is a pretty niche add.